src/desktop/apps/authentication/__tests__/loginRoute.jest.ts

```diff
@@ -1,4 +1,6 @@
-import { FullPageAuthStatic } from "../components/FullPageAuthStatic"
+/* eslint-disable jest/no-done-callback */
+
+import { FullPageAuthStatic } from "v2/Apps/Authentication/Components/FullPageAuthStatic"
```

Comment:
I'd probably rename this to something more sensible

Reply:
I didn't want to mess with any naming because this stuff is used elsewhere. I was gonna do an auth form cleanup after this PR, which basically just moves stuff and rearranges a bit.